lib/backup/azremote/azblob.go

```diff
@@ -178,12 +178,35 @@ func (fs *FS) CopyPart(srcFS common.OriginFS, p common.Part) error {
 		return fmt.Errorf("failed to generate SAS token of src %q: %w", p.Path, err)
 	}
 
-	// Hotfix for SDK issue: https://github.com/Azure/azure-sdk-for-go/issues/19245
-	t = strings.Replace(t, "/?", "?", -1)
 	ctx := context.Background()
-	_, err = dbc.CopyFromURL(ctx, t, &blob.CopyFromURLOptions{})
+
+	// In order to support copy of files larger than 256MB, we need to use the async copy
+	// Ref: https://learn.microsoft.com/en-us/rest/api/storageservices/copy-blob-from-url
+	_, err = dbc.StartCopyFromURL(ctx, t, &blob.StartCopyFromURLOptions{})
```

Comment:
should we use it only for parts > 256MB? and use sync method for parts up to 256mb?

Reply:
It seems to me like using single flow for all files should better in terms of code complexity and will be easier to maintain.
Implementing 2 methods might require us updating code when/if azure limits for blocking method will change, while always using async copy won't require any additional changes.